terraform/modules/concourse_web/data.tf

```diff
@@ -18,10 +18,10 @@ data "aws_ssm_parameter" "concourse_password" {
   name = var.concourse_secrets.password_ssm_name
 }
 
-data "aws_ssm_parameter" "concourse_cognito_client_id" {
-  name = var.cognito.client_id_ssm_parameter
-}
-
-data "aws_ssm_parameter" "concourse_cognito_client_secret" {
-  name = var.cognito.client_secret_ssm_parameter
-}
+//data "aws_ssm_parameter" "concourse_cognito_client_id" {
+//  name = var.cognito.client_id_ssm_parameter
+//}
+//
+//data "aws_ssm_parameter" "concourse_cognito_client_secret" {
+//  name = var.cognito.client_secret_ssm_parameter
+//}
```

Comment:
Should we remove this commented code?

Reply:
I think they should stay as we intend to do the work shortly